project/frontend/src/components/SGonksLists/ShortSGonksList/ShortSGonksList.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import classes from "./ShortSGonksList.module.css";
+
+import ShortSGonksRow from "./ShortSGonksRow/ShortSGonksRow";
+import { Link } from "react-router-dom";
+
+const ShortSGonksList = (props) => {
+  console.log(props.sgonks);
+  const sGonksList = props.sgonks.slice(0, 7).map((singularSGonk) => {
+    return (
+      <ShortSGonksRow
+        sgonk={singularSGonk}
+        key={singularSGonk.searchTerm}
+      ></ShortSGonksRow>
+    );
+  });
+
+  return (
+    <div {...props} className={classes.List}>
+      {sGonksList}
+      {props.sgonks.length > 7 ? (
```

Comment:
What is the purpose of 7?  Can you make it into a constant and the constant name can explain what the purpose is

Reply:
Good idea.

Reply:
Done.